camera_error_handler.py

```diff
@@ -0,0 +1,34 @@
+"""
+Exception handler for the Raspicam 
+
+Used alongside the 'with' magic Python word.
+eg. with CameraException(self.client, self.backend_name)
+"""
+from json import dumps
+from traceback import format_exc
+
+from config import read_configs
+from topics import Camera
+
+class CameraException:
+    def __init__(self, client, backend:str):
+        self.configs = read_configs()
+        self.client = client
+        self.camera = self.configs["device"]
+        self.backend = backend
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, type, value, traceback):
```

Comment:
`type` isn't a good variable name, as `type` is also a python built-in. By naming a variable this, we're unable to use the python built-in. Consider something like `exc_type` or `exception_type`.